src/configuration/cloudformation.coffee

```diff
@@ -48,9 +49,23 @@ apiConfig = async (dir, globals) ->
   yield preprocessors.api mungedConfig
 
 renderAPI = async (dir, globals) ->
+  H = require "handlebars"
   mungedConfig = yield apiConfig dir, globals
-  template = yield read resolve skyMixinsPath, "api.yaml"
-  yaml _render template, mungedConfig
+  # TODO: factor this out into something saner
```

Comment:
I still feel more comfortable working within Sky for a while, to avoid the complications of working on numerous repos.